projects/plugins/jetpack/tests/e2e/lib/pages/homepage.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * Internal dependencies
+ */
+import WpPage from './wp-page';
+import logger from '../logger';
+import { searchResultForTest1, searchResultForTest2 } from '../search-helper';
+
+export default class Homepage extends WpPage {
+	static SEARCH_API_PATTERN = /^https:\/\/public-api\.wordpress.com\/rest\/v1.3\/sites\/\d+\/search.*/;
+	static WAIT_FOR_ANIMATION_AND_RENDERING_TIMEOUT = 2000;
+
+	constructor( page ) {
+		const url = `${ siteUrl }/`;
+		super( page, { expectedSelectors: [ '.site-title' ], url } );
+	}
+
+	async registerRouteInterceptions() {
+		await this.searchAPIRoute();
+	}
+
+	searchAPIRoute() {
```

Comment:
Could do with a docblock or some comments here explaining what this does.

Reply:
added in https://github.com/Automattic/jetpack/pull/19447/commits/9a282cad876e146369da6936770672802c493bb6